File: liveDemo/Inherit.js
swiftFile = Polyglot.evalFile("llvm", "Inherit");

parentObj = swiftFile.ObjectCreator.requestParent()
childObj = swiftFile.ObjectCreator.requestChild()


print("16 ", parentObj.square(4))
print("25 ", childObj.square(4))
print("3.5", parentObj.get3P5())
print("3.5", childObj.get3P5())

File: liveDemo/Inherit.swift
public class Parent {
	
	public func square(n: Int) -> Int {
		return n*n
	}
	
	public func get3P5() -> Double {
		return 3.5;
	}
}

public class Child: Parent {
	public override func square(n: Int) -> Int {
		return (n+1)*(n+1)
	}
}


public class ObjectCreator {
	public static func requestParent() -> Parent {
		return Parent()
	} 
	
	public static func requestChild() -> Parent {
		return Child()
	} 
}